lib/matplotlib/table.py

```diff
@@ -461,8 +456,7 @@ def auto_set_column_width(self, col):
         self.stale = True
 
     def _auto_set_column_width(self, col, renderer):
-        """ Automagically set width for column.
-        """
+        """Automatically set width for column."""
```

Comment:
I think automagically is fine here, we're allowed some humor for private docstrings at least...

Reply:
Not sure it was intended, though. All other similar methods around this are using "automatically" and 'g' and 't' are adjacent on the keyboard, so I assume this is just a funny typo.

Reply:
I don't really care tbh, changing is fine.